app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryViewCallback.kt

```diff
@@ -15,30 +15,24 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
-package org.kiwix.kiwixmobile.zim_manager.library_view;
 
-import java.util.LinkedList;
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity;
-import org.kiwix.kiwixmobile.core.entity.LibraryNetworkEntity.Book;
+package org.kiwix.kiwixmobile.zimManager.libraryView
```

Comment:
@anamansari062 Also you need to change the package name `zimManager`and `librayView`.
![Screen Shot 2021-11-30 at 22 51 35](https://user-images.githubusercontent.com/43576162/144096338-d4959994-67bc-4a62-8964-70d44bf4bea9.png)

Reply:
@gouri-panda sure i will do the changes